tools/python/boutiques/schema/descriptor.schema.json

```diff
@@ -0,0 +1,501 @@
+{
```

Comment:
I reverted the files and redid this move, ensuring I did `git mv` - it appears like no dice locally, but maybe when it's merged the history changes will have taken effect.